frontend/benefit/applicant/src/components/applications/forms/application/step1/companyInfo/CompanyInfo.sc.ts

```diff
@@ -0,0 +1,66 @@
+import { Notification as HDSNotification } from 'hds-react';
+import { StyledFormGroup } from 'shared/components/forms/section/styled';
+import styled from 'styled-components';
+
+const CompanyInfoContainer = styled.div`
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-areas:
+    'info notification'
+    'address address'
+    'iban iban';
+  width: 100%;
+`;
+
+const CompanyInfoSection = styled.div`
+  grid-area: info;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const CompanyInfoColumn = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 50%;
+`;
+
+const CompanyInfoRow = styled.div<{ $loading?: boolean }>`
+  display: flex;
+  line-height: ${(props) => props.theme.lineHeight.l};
+  height: ${(props) => `calc(${props.theme.lineHeight.l} * 1em)`};
+  background-color: ${({ $loading, theme }) =>
+    $loading ? theme.colors.black10 : 'inherit'};
```

Comment:
This is quite simple yet working idea for loading to use different css styles. I had the same loading feature but I used react-loading-skeleton instead: https://github.com/dvtng/react-loading-skeleton to have animations as well but I think both solutions are good.

Reply:
One thing to note is accessiblity, altering the css only would not give any feeback for screen reader that the loading is in progress. But if the hds LoadingSpinner is visible at the same time (on FormSection), it gives the feedback so this is ok in accessibility point of view.